### PR TITLE
atlasexec/migrate-lint: added --base param

### PR DESCRIPTION
**Pull request by dorav**

### CODE REVIEW

File: atlasexec/atlas_test.go [hunk 1]
```diff
@@ -134,6 +136,19 @@ func TestMigrateLint(t *testing.T) {
 		var raw json.RawMessage
 		require.NoError(t, json.NewDecoder(&buf).Decode(&raw))
 	})
+	t.Run("lint uses --base and --latest", func(t *testing.T) {
+		c, err := atlasexec.NewClient(".", "atlas")
+		require.NoError(t, err)
+		summary, err := c.MigrateLint(context.Background(), &atlasexec.MigrateLintParams{
+			DevURL: "sqlite://file?mode=memory",
+			DirURL: "file://testdata/migrations",
+
```
Comment (a8m) [inline atlasexec/atlas_test.go:145]:
```suggestion
```

File: atlasexec/atlas_test.go [hunk 2]
```diff
@@ -201,6 +230,19 @@ func TestMigrateLintWithLogin(t *testing.T) {
 		require.NoError(t, err)
 		require.Equal(t, strings.TrimSpace(string(buf.Bytes())), "https://migration-lint-report-url")
 	})
+	t.Run("lint uses --base", func(t *testing.T) {
+		c, err := atlasexec.NewClient(".", "atlas")
+		require.NoError(t, err)
+		summary, err := c.MigrateLint(context.Background(), &atlasexec.MigrateLintParams{
+			DevURL: "sqlite://file?mode=memory",
+			DirURL: "file://testdata/migrations",
+
```
Comment (a8m) [inline atlasexec/atlas_test.go:239]:
```suggestion
```

File: atlasexec/atlas_test.go [hunk 3]
```diff
@@ -235,6 +277,18 @@ func generateHCL(t *testing.T, url, token string) string {
 	return atlasConfigURL
 }
 
+func testDir(t *testing.T, path string) (d migrate.MemDir) {
```
Comment (a8m) [inline atlasexec/atlas_test.go:280]: See migrate.LocalDir

File: atlasexec/atlas_test.go [hunk 4]
```diff
@@ -161,6 +177,19 @@ func TestMigrateLintWithLogin(t *testing.T) {
 		switch {
 		case strings.Contains(query.Query, "mutation reportMigrationLint"):
 			fmt.Fprintf(w, `{ "data": { "reportMigrationLint": { "url": "https://migration-lint-report-url" } } }`)
+		case strings.Contains(query.Query, "query dirs"):
+			dir := testDir(t, "./testdata/migrations")
```
[the remark this answers is not in the document]
Comment (dorav) [reply atlasexec/atlas_test.go:181]: Didn't know about it, this was copied from a different UT

File: atlasexec/atlas_test.go [hunk 5]
```diff
@@ -1,9 +1,11 @@
 package atlasexec_test
 
 import (
+	"ariga.io/atlas/sql/migrate"
```
Comment (a8m) [inline atlasexec/atlas_test.go:4]: move this one to the other group

**Review by a8m — APPROVED** [review]
Approved. See the goimports comment

**Pull request merged**